sagemaker-pipeline-multi-model/sagemaker-pipeline/pipelines/restate/pipeline.py

```diff
@@ -166,14 +142,14 @@ def get_pipeline(
     # You can configure this with your own bucket name, e.g.
     # bucket = "my-bucket"
     bucket = sess.default_bucket()
+
     print(f"Data Wrangler export storage bucket: {bucket}")
 
     # unique flow export ID
     flow_export_id = f"{time.strftime('%d-%H-%M-%S', time.gmtime())}-{str(uuid.uuid4())[:8]}"
     flow_export_name = f"flow-{flow_export_id}"
 
     # Output name is auto-generated from the select node's ID + output name from the flow file.
-    # output_name = "dc0bda28-f867-4501-9323-b3d571ae5c35.default"
     output_name = "99ae1ec3-dd5f-453c-bfae-721dac423cd7.default"
```

Comment:
Should this be obfuscated/deleted? Does the customer need to fill it in with their own output name?

Reply:
This output_name was derived from the the notebook generated from  [Data Wrangler Export -> Jupyter Notebook to Create a Pipeline](https://docs.aws.amazon.com/sagemaker/latest/dg/data-wrangler-data-export.html). This output_name works with the specific flow, in this case is restate-athena-california.flow. '99ae1ec3-dd5f-453c-bfae-721dac423cd7' is a Node ID in the Data Wrangler flow file. Thank you!